server/src/categories/dto/create-category.dto.ts

```diff
@@ -1,12 +1,16 @@
+import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
 import { IsNotEmpty, IsOptional } from 'class-validator';
 
 export class CreateCategoryDto {
+  @ApiProperty()
   @IsNotEmpty()
   name: string;
 
+  @ApiPropertyOptional()
   @IsOptional()
   applies_to_assets: boolean;
 
+  @ApiPropertyOptional()
   @IsOptional()
   applies_to_organizations: boolean;
```

Comment:
You can get the same behavior by removing the annotation and adding the optional property denotator:

```suggestion
  @IsOptional()
  applies_to_assets?: boolean;

  @IsOptional()
  applies_to_organizations?: boolean;
```